lib/obsolete.gd

```diff
@@ -682,6 +681,19 @@ DeclareObsoleteSynonym( "RecFields", "RecNames" );
 ##  Not used in any redistributed package (11/2018)
 #DeclareObsoleteSynonym( "SHALLOW_SIZE", "SIZE_OBJ" );
 
+
+#############################################################################
+##
+#V  InfoRead? 
+##
+##  InfoRead used to be used to print when a file is read using `Read()`
+##  
+##  These variables are still referred to by atlasrep, gbnp, guava, hecke
+##  and xgap.
```

Comment:
@markuspf please update this comment - only atlasrep and gbnp remain (hecke and xgap were done today by @fingolfin).